src/controllers/legacycontrollermappingfilehandler.h

```diff
@@ -67,6 +72,13 @@ class LegacyControllerMappingFileHandler {
 
     bool writeDocument(const QDomDocument& root, const QString& fileName) const;
 
+#ifdef MIXXX_USE_QML
+    // Maximum target frame per request for a a screen controller
+    static constexpr int s_maxTargetFps = 240;
+    // Maximum time allowed for a screen to run a splash off animation
+    static constexpr int s_maxSplashOffDuration = 3000;
```

Comment:
this feels like an extraneous unnecessary option, but I'm ok to leave it. (Sometimes it's ok to pick a number and not let the user change it :))

Reply:
It is useful to allow the screen to "clean" themselves up (on the S4 Mk3, this means displaying the Mixxx logo before stopping). Without a Splash Off feature, the screen will look like there are crashed.
Arguably, I could have made a "last cleaning frame" instead, but thought this could allow some fancy animation to make the UX even smoother